infra/auth-logic/src/main/java/com/simprints/infra/authlogic/authenticator/Authenticator.kt

```diff
@@ -45,6 +46,8 @@ internal class Authenticator @Inject constructor(
             projectAuthenticator.authenticate(nonceScope, projectSecret)
 
             logMessageForCrashReportWithNetworkTrigger("Sign in success")
+            authStore.signedInUserId = userId
```

Comment:
Should you not store the tokenized version of the userId same as in the `UpdateStoredUserIdUseCase` as you will need the tokenized version for the down-sync?

Reply:
From what I looked at, the userId is never tokenised when we are saving it (on login, it comes from the form as is; on intent, it comes from params always as `Tokenisable.Raw`).

Reply:
It looks like you are tokenizing the user id in the different request builder

Reply:
Hmm, let me rewrite this stuff to save the tokenised string all the way down.